lib/views/widgets/color_picker_keyboard.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
```

Comment:
<img width="336" alt="Screen Shot 2022-10-19 at 14 28 48" src="https://user-images.githubusercontent.com/107173849/196625084-5ee0e429-788d-4044-b9bc-36bde1be1dc5.png">

It seems to be wrong compared to the design. Can you recheck it?

Reply:
IMO, it's not so important. It's just different layout.

Reply:
I updated it. Please check again. @ManhNTX 

![Simulator Screen Shot - iPhone 14 Pro Max - 2022-10-20 at 11 14 52](https://user-images.githubusercontent.com/80730648/196854972-4c3a4788-8c43-46b2-8fce-31e75a6a2429.png)